ibm/mas_devops/roles/appconnect/README.md

```diff
@@ -14,31 +14,31 @@ Optional - Defines the targetted cluster namespace/project where AppConnect will
 - Default Value: `ibm-app-connect`
 
 ### appconnect_channel
-Optional - AppConnect subscription channel to be installed. For MAS 8.7, default `v3.0` channel is used. Ensure the supported dashboard version and license ID according to the AppConnect operator/channel version.
+Optional - AppConnect subscription channel to be installed.
+Ensure the supported dashboard version and license ID according to the AppConnect operator/channel version.
 
-- MAS 8.4 uses AppConnect channel subscription v1.4 and license id = L-APEH-BSVCHU
-- MAS 8.5 uses AppConnect channel subscription v2.0 and license id = L-KSBM-BZWEAT
 - MAS 8.6 uses AppConnect channel subscription v2.0 and license id = L-KSBM-C37J2R
 - MAS 8.7 uses AppConnect channel subscription v3.0 and license id = L-KSBM-C87FU2
+- MAS 8.8 uses AppConnect channel subscription v4.2 and license id = L-APEH-C9NCK6
```

Comment:
See comments in Slack ... do we really need to tied AppConnect channel/version 1:1 to MAS channel/version?

Reply:
Waiting on response from https://ibm-watson-iot.slack.com/archives/C0195MVCEUD/p1661865932206939?thread_ts=1661842890.671729&cid=C0195MVCEUD

The default currently is using:

```
appconnect_channel: "{{ lookup('env', 'APPCONNECT_CHANNEL') | default('v4.2', true) }}"
appconnect_license_id: "{{ lookup('env', 'APPCONNECT_LICENSE_ID') | default('L-APEH-C9NCK6', true) }}"
```

Reply:
@durera can we consider it ok given the response from HPU team?